exercises/react/tests/react.rs

```diff
@@ -101,32 +101,29 @@ fn error_setting_a_compute_cell() {
 
 use std::cell::RefCell;
 struct Expector {
-    have_value: RefCell<bool>,
-    value: RefCell<isize>,
+    value: RefCell<Option<isize>>,
 }
 
 impl Expector {
     fn new() -> Self {
         Expector {
-            have_value: RefCell::new(false),
-            value: RefCell::new(0),
+            value: RefCell::new(None),
         }
     }
 
     fn expect_to_have_been_called_with(&self, v: isize) {
-        assert!(*self.have_value.borrow(), "Callback was not called, but should have been");
-        assert_eq!(*self.value.borrow(), v, "Callback was called with incorrect value");
-        *self.have_value.borrow_mut() = false;
+        assert_ne!(*self.value.borrow(), None, "Callback was not called, but should have been");
+        assert_eq!(*self.value.borrow(), Some(v), "Callback was called with incorrect value");
```

Comment:
I'd consider using https://doc.rust-lang.org/std/cell/struct.RefCell.html#method.replace in this line to remove the need for the line below it.

Reply:
Yes, I think that would be natural.